core/ulog/log_test.go

```diff
@@ -134,6 +134,24 @@ func TestConfiguredLoggerWithTextFormatter(t *testing.T) {
 	})
 }
 
+func TestConfiguredLoggerWithTextFormatter_NonDev(t *testing.T) {
+	dev := development
+	development = false
```

Comment:
`development` should probably be a `_development` global to indicate it's a global, right?